tests/test_config_section_libvirtd.py

```diff
@@ -102,6 +102,47 @@ def test_validate_server_full_url(self):
         result = self.virt_config._validate_server('server')
         self.assertIsNone(result)
 
+    def test_validate_uuid_hypervisor_id(self):
+        """
+        Test validation of valid hypervisor_id
+        """
+        # Following method sets hypervisor_id to 'uuid' and it is valid
+        self.init_virt_config_section()
+        result = self.virt_config._validate_hypervisor_id('hypervisor_id')
+        self.assertIsNone(result)
+
+    def test_validate_hostname_hypervisor_id(self):
+        """
+        Test validation of valid hypervisor_id
+        """
+        self.init_virt_config_section()
+        self.virt_config['hypervisor_id'] = 'hostname'  # it is also valid value
+        result = self.virt_config._validate_hypervisor_id('hypervisor_id')
+        self.assertIsNone(result)
+
+    def test_validate_unvalid_hypervisor_id(self):
+        """
+        Test validation of valid hypervisor_id
+        """
+        self.init_virt_config_section()
+        self.virt_config['hypervisor_id'] = 'unsupported_id'
+        result = self.virt_config._validate_hypervisor_id('hypervisor_id')
+        expected_result = [
+            ('warning', 'Invalid option: "unsupported_id" for hypervisor_id, use one of: (uuid, hostname)')
```

Comment:
This should be 'error' level as it invalidates the configuration to use an unsupported_id.
The configuration needs a valid hypervisor_id and so it should be considered invalid if a bad one is given.